services/shuttle-poise/src/lib.rs

```diff
@@ -72,5 +72,15 @@ impl<T, E> From<Arc<poise::Framework<T, E>>> for PoiseService<T, E> {
     }
 }
 
-/// The return type that should be returned from the [shuttle_runtime::main] function.
+/// Return type from the `[shuttle_runtime::main]` macro for a Poise-based service.
+///
+/// # Example
+///
+/// ```rust,no_run
+/// #[shuttle_runtime::main]
+/// async example_service() ->
+///   ShuttlePoise<impl FnOnce(&mut ServiceConfig) + Send + Clone + 'static> {
+///   todo!()
+/// }
```

Comment:
ditto. Please refer to: https://github.com/shuttle-hq/shuttle-examples/blob/main/poise/hello-world/src/main.rs#L18.